contracts/openzeppelin/Ownable.sol

```diff
@@ -13,6 +13,7 @@ import "./Context.sol";
  */
 contract Ownable is Context {
     address private _owner;
+    mapping (address => bool) private _adminMap;
```

Comment:
contracts under openzeppelin folder were copied from openzeppeline and untouched. better not to change it because it will create a surprise for readers.

Reply:
@aiolos404 let's create a new contract lib on top of Ownable to avoid touching forked codes.

Reply:
Will do